doc/source/example_notebooks/IS2_data_read-in.ipynb

```diff
@@ -63,9 +63,8 @@
    "metadata": {},
```

Comment:
<p>Remove statement that cloud data requires beta tester access</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icesat2py/icepyx/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>